app/controllers/answers_controller.rb

```diff
@@ -58,6 +65,10 @@ def destroy
 
   private
     # Use callbacks to share common setup or constraints between actions.
+    def set_question
+      @answer = Question.find(params[:question_id])
```

Comment:
I'm not sure it's about finding the answer or the question.

Reply:
I think you want to write it like this with **memoization**

`@question ||= Question.find(params[:question_id])`